common/src/main/java/io/github/strikerrocker/cleardespawn/IConfigHelper.java

```diff
@@ -1,9 +1,16 @@
 package io.github.strikerrocker.cleardespawn;
 
+import net.minecraft.world.entity.item.ItemEntity;
+
 public interface IConfigHelper {
     int getFlashStartTime();
 
+    @Deprecated(forRemoval = true)
     int getItemDespawnTime();
 
+    default int getItemDespawnTime(ItemEntity entity) {
```

Comment:
Not sure why you overload the method Just replace the original method

Reply:
>The configuration options in Forge have been deprecated for removal but not removed to maintain compatibility.

But ok, I can do what you said. I generally avoid making binary breaks in public API when I think someone else may be using it.